### PR TITLE
add breadcrumb separator variable

### PR DESCRIPTION
**Pull request by bitbruder**
Adds variable for custom breadcrumb separators instead of a slash.

Reference #8131

### CODE REVIEW

**Review by IamManchanda — APPROVED** [review]
LGTM...

**Comment by IamManchanda** [comment]
Checked it out locally... This is looking good...
@kball for merge/review!

**Comment by IamManchanda** [comment]
@digitalfrood... By the way congrats on your first time contribution ;)

File: scss/components/_breadcrumbs.scss [hunk 1]
```diff
@@ -34,13 +34,21 @@ $breadcrumbs-item-margin: 0.75rem !default;
 /// @type Boolean
 $breadcrumbs-item-uppercase: true !default;
 
-/// If `true`, adds a slash between breadcrumb links.
+/// If `true`, adds a seperator between breadcrumb links.
 /// @type Boolean
-$breadcrumbs-item-slash: true !default;
+$breadcrumbs-item-separator: true !default;
```
Comment (kball) [inline scss/components/_breadcrumbs.scss]: Can we do a check for if `$breadcrums-item-slash` is defined and use that if so? That would enable backwards compat with older settings files

**Comment by kball** [comment]
Let 1 item of feedback, other than that LGTM, can we get that addressed and then will merge?

File: scss/components/_breadcrumbs.scss [hunk 2]
```diff
@@ -34,13 +34,21 @@ $breadcrumbs-item-margin: 0.75rem !default;
...
+$breadcrumbs-item-separator-item: '/' !default;
+
+/// Used character for the breadcrumb separator in rtl mode.
+/// @type Content
+$breadcrumbs-item-separator-item-rtl: '\\' !default;
+
+/// Color of breadcrumb item.
 /// @type Color
-$breadcrumbs-item-slash-color: $medium-gray !default;
+$breadcrumbs-item-separator-color: $medium-gray !default;
```
Comment (IamManchanda) [inline scss/components/_breadcrumbs.scss]: Same for this also @kball 

Again warn @digitalfrood i am taking this up!

**Comment by IamManchanda** [comment]
done @kball ... Anything else needed or good to go??

**Pull request merged**